__tests__/books.test.js

```diff
@@ -0,0 +1,63 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const { Book } = require('../lib/models/Books');
+
+
+describe('books routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+  it('/books/:id should return a list of book by id with author', async() => {
+    const res = await request(app).get('/books/1');
+    expect(res.body).toEqual({    
+      'id': expect.any(String),
```

Comment:
When testing and using these `expect.any()` statements I think it's always good to be specific when we can. when you're getting a specific id this is one of those instances where we can pretty easily get specific since you are giving it that id. So I would suggest expecting the id to be '1' rather than `expect.any(String)`. If we expect anything there and are for some reason not getting 1 we won't know it in the test.